### PR TITLE
Remove Cassandra

### PR DESCRIPTION
**Pull request by wjhoward**
Decommissioning the Cassandra datastore code and tests.

### CODE REVIEW

**Review by itsgc — APPROVED** [review]
With a grain of salt but LGTM!

File: itest/test/cassandra/timeouts_test.py [hunk 1]
```diff
@@ -1,23 +0,0 @@
...
-            duration = time.time() - start
-
-            # Cassandra timeouts are treated as a miss.
-            assert response.headers['Spectre-Cache-Status'] == 'miss'
-
-            # The timeouts are set at 500ms in itests.
-            # Duration should be very close to that value. If it's much higher it
-            # means the timeouts are not working, if it's lower it means the
-            # iptable rule is not working.
-            assert duration - 0.5 < 0.01
```
Comment (khvzak) [inline itest/test/cassandra/timeouts_test.py:23]: Should we keep this to test redis timeouts?

**Review by khvzak — APPROVED** [review]
LGTM!

File: itest/test/cassandra/timeouts_test.py (hunk 1 above)
Comment (wjhoward) [reply itest/test/cassandra/timeouts_test.py:23]: I was also considering this, will add it back in.

Comment (wjhoward) [reply itest/test/cassandra/timeouts_test.py:23]: Following further investigation, I noticed that this test wasn't even being collected and run due to the folder structure, and if you do actually run it, it fails. I do not think the test is fit for Redis, and we can add some more appropriate tests when we later refactor the tests.

**Pull request merged**